didx-xyz/yoma-mobile
# YOMA-495 - feat(CvCard): adjust to handle children

## Scope [YOMA-495](https://yomaworld.atlassian.net/browse/YOMA-495)
- we need to adjust Cv Card to be able to take children.

![](https://media.giphy.com/media/PjsRHq04WuRk8U4uJo/giphy.gif)
## Work Done
- added children to CvCard
- cleaned up the component somewhat, and pulled out some dependency components
- added a `rightAccessory` prop to `ListCardHeader` so we can pass the Edit button directly into the card header.

## Steps to Test
- navigate to MyCV / DigitalCv and make sure it looks ok
- ensure edit buttons work and text etc looks correct
- also make sure that the My Skills view looks fine (the other place that `ListCardHeader` is being used.
